test/com/facebook/buck/testrunner/testdata/simple_testng_setup/test/SimpleTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2014-present Facebook, Inc.
```

Comment:
All files *must* contain this license preamble - it's actually checked within their codebase by one of their unit tests